lib/validateKey.js

```diff
@@ -15,38 +17,51 @@ var SPAM = "Personalisation Algorithm are a collective issue, and only collectiv
 var M = "link my user to this key: ";
 
 function validateKey(req) {
-    var postId = _.parseInt(req.params.postId);
-    debug("This is the postId where we expect to find a key %d", postId);
+    var permalink = req.body.permalink;
+    var publicKey = req.body.publicKey;
+    console.log("Looking for publicKey %s in %s", publicKey, permalink);
     return request
         .getAsync({
-            'url': 'https://www.facebook.com/' + postId,
+            'url': permalink,
             'headers': {
                 'User-Agent': "User-Agent: curl/7.47.0"
             }
         })
         .then(function(page) {
-            debugger;
-            var mndx = page.body.indexOf(M);
-            if(_.eq(mndx, -1)) {
+            var mndx = page.body.indexOf(publicKey);
+            fs.writeFileSync('/home/vrde/repos/ESCVI/page.html', page.body);
```

Comment:
`fs` is wrapped in Bluebird promises, probably I'll review to use writeFileAsync (do not exists in the `fs` module but exists after the promisifyAll)